project_national/national_upgrades.yml

```diff
@@ -65,6 +65,20 @@ upgrades:
             multiplier: Wall Area, Above-Grade, Conditioned (ft^2)
         lifetime: 999
 
+  - upgrade_name: Foundation Type
+    options:
+    - option: Geometry Foundation Type|Unvented Crawlspace
+      apply_logic:
+        and:
+          - Geometry Foundation Type|Vented Crawlspace
+          - or:
+              - Geometry Building Level MF|None
+              - Geometry Building Level MF|Bottom
```

Comment:
I remember you talking about this but don't recall the details. Why is this needed again? Seems confusing and easy for a user to miss.

Reply:
Because of [this](https://github.com/NREL/resstock/blob/develop/measures/ResStockArguments/measure.rb#L700-L712). You can sample a crawlspace for a "middle" or "top" apartment, but the foundation type gets changed to "above apartment" (not actually a crawlspace).

Reply:
Oh right, it's because ResStock is [still defining foundations on the building level, not the unit level](https://github.com/NREL/resstock/issues/728). ResStock users must get confused often about this.